homework/App.js

```diff
@@ -12,18 +12,31 @@ class App {
    * @param {string} url The GitHub URL for obtaining the organization's repositories.
    */
   async initialize(url) {
-    // Add code here to initialize your app
-    // 1. Create the fixed HTML elements of your page
-    // 2. Make an initial XMLHttpRequest using Util.fetchJSON() to populate your <select> element
-
-    const root = document.getElementById('root');
-
-    Util.createAndAppend('h1', root, { text: 'It works!' }); // TODO: replace with your own code
-
     try {
+      const root = document.getElementById('root');
+      const header = Util.createAndAppend('header', root, { class: 'header' });
+      Util.createAndAppend('div', root, {
+        class: 'container',
+        id: 'container',
+      });
+      const headerDiv = Util.createAndAppend('div', header, { class: 'headerDiv' });
```

Comment:
You could add an `autofocus` attribute here so that the select element get the keyboard focus when the page is first loaded.  A visually impaired user can then immediately change the selected repo by using up and down keys.